internal/isoutil/isoutil.go

```diff
@@ -327,3 +379,14 @@ func writeOffsetInfo(headerOffset int64, offsetInfo *OffsetInfo, iso *os.File) (
 
 	return headerOffset, nil
 }
+
+func readOffsetInfo(headerBytes []byte) (*OffsetInfo, error) {
```

Comment:
Same here, all this header stuff should be in the editor.

Reply:
moved all :)